### PR TITLE
Check if repository is available before configuring it

### PR DESCRIPTION
**Pull request by racciari**

### CODE REVIEW

File: tasks/Debian.yml [hunk 1]
```diff
@@ -1,4 +1,15 @@
 ---
+- name: Check if the repository is available
+  uri:
+    url: "{{ openio_repository_mirror_url_base }}/{{ repo.key }}/{{ repo.value.release }}/{{ ansible_lsb.id | default(ansible_distribution)}}/{{ ansible_distribution_release }}"
+  with_dict: "{{ openio_repository_products }}"
+  when:
+    - (repo.value.state is not defined or repo.value.state != 'absent')
```
Comment (vincent-legoll) [inline tasks/Debian.yml]: useless parenthesis

Comment (racciari) [reply tasks/Debian.yml]: Removed

Comment (cdelgehier) [reply tasks/Debian.yml]: ping @racciari

Comment (racciari) [reply tasks/Debian.yml]: ?

File: tasks/Ubuntu.yml [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+- name: Check if the repository is available
+  uri:
+    url: "{{ openio_repository_mirror_url_base }}/{{ repo.key }}/{{ repo.value.release }}/{{ ansible_lsb.id | default(ansible_distribution)}}/{{ ansible_distribution_release }}"
+  with_dict: "{{ openio_repository_products }}"
+  when:
+    - repo.value.state is not defined or repo.value.state != 'absent'
+    - openio_repository_check_reachability
+  loop_control:
+    loop_var: repo
+#  no_log: '{{ openio_repository_no_log }}'
```
Comment (vincent-legoll) [inline tasks/Ubuntu.yml]: Why is this one commented ?

Comment (racciari) [reply tasks/Ubuntu.yml]: For testing purposes, thanks it's fixed!

**Pull request merged**